package.json

```diff
@@ -56,10 +56,10 @@
     "@vue/compiler-dom": "3.5.13",
     "@vue/shared": "3.5.13",
     "c12": "3.0.2",
-    "h3": "npm:h3-nightly@1.15.1-20250222-111608-d1c00fc",
+    "h3": "1.15.1",
```

Comment:
nitro v3 used h3-v2 ? `[2.0.0-20250312-144942-506a98f](https://www.npmjs.com/package/h3-nightly/v/2.0.0-20250312-144942-506a98f)` or no ?

Reply:
not yet